tests/unit/disco/components/TestAddon.js

```diff
@@ -143,22 +141,31 @@ describe('<Addon />', () => {
       const data = { ...result, needsRestart: true, status: UNINSTALLING };
       const root = renderAddon({ addon: data, ...data });
       const restart = findDOMNode(root).querySelector('.notification.restart');
-      expect(restart.querySelector('p').textContent).toEqual('This add-on will be uninstalled after you restart Firefox.');
+
+      expect(restart.querySelector('p').textContent).toEqual(
+        'This add-on will be uninstalled after you restart Firefox.');
     });
 
     it('does not normally render a restart notification', () => {
       const root = renderAddon({ addon: result, ...result });
-      expect(findDOMNode(root).querySelector('.notification.restart')).toBeFalsy();
+
+      expect(findDOMNode(root).querySelector('.notification.restart'))
+        .toBeFalsy();
     });
 
     it('renders the heading', () => {
       const root = renderAddon({ addon: result, ...result });
-      expect(root.heading.textContent).toContain('test-heading');
+
+      expect(findDOMNode(root).querySelector('.heading').textContent)
```

Comment:
It is, and we'll be moving to enzyme, but I didn't want to refactor these tests at the time. I could do it now if you like but this isn't our only usage of `findDomNode` 😄